### DISCUSSION TITLE
is REMOTE Hardware Transcoding possible?

### DISCUSSION THREAD

**Discussion by streetcap** [immich-app/immich#5585] [Q&A]
With the release of the new experimental feature of Hardware Transcoding (https://immich.app/docs/features/hardware-transcoding), I was wondering if it is possible to hand off the hardware transcoding to a remote machine (similar to how the ML can be processed remotely: https://immich.app/docs/guides/machine-learning).

My transcoding jobs is at 250,000 so it will probably be transcoding long after I am dead.  lol.

I am running immich on an old NAS but have a 12th gen intel that would be infinitely better at processing these.

**Comment by bo0tzz** [comment]
Yes, see #3026. You can just spin up an extra microservices container on another host.

**Comment by erkexzcx** [comment]
For those coming via Google, I wrote guide here: https://github.com/immich-app/immich/discussions/14142
